scripts/cms-run-migrations.ts

```diff
@@ -0,0 +1,8 @@
+import { execCommand } from './lib/exec-command';
+import { datocmsEnvironment } from '../datocms-environment';
+
+const confirmationMessage = `Apply all migrations in chronological order to the environment '${datocmsEnvironment}'`;
+execCommand(
```

Comment:
What about an option to run a migration `--in-place`?
Let a developer select if it needs to be in-place or with creating a new env.
```js
if (migrationType === 'in-place') {
    return await execCommandSafe(
      `npx datocms migrations:run --source=${targetEnvironment} --in-place`,
    );
  } else {
    return await execCommandSafe(
      ` npx datocms migrations:run --destination=${targetEnvironment}`,
    );
  }

Reply:
I definitely see the need to add an option for `--in-place` but I'd rather put it in a separate script and try to fit it in the defined workflow.